opencog/query/InitiateSearchCB.cc

```diff
@@ -761,7 +761,7 @@ bool InitiateSearchCB::variable_search(PatternMatchEngine *pme)
 			if (prev != _pattern) { prev = _pattern; count = 0; }
 			else {
 				count++;
-				if (5 < count)
+				if (100 < count)
 					throw RuntimeException(TRACE_INFO,
```

Comment:
OK, but if you are hitting this more than 1x, you've got a bug.  I thought 5x was overkill ... so 100x makes no sense at all, I don't think.  Or am I missing something?